packages/react-devtools-shared/src/devtools/views/Components/reach-ui/tooltip.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @flow
+ */
+
+import * as React from 'react';
+import ReachTooltip from '@reach/tooltip';
+import useThemeStyles from '../../useThemeStyles';
+
+const Tooltip = ({children, ...props}) => {
+  const style = useThemeStyles();
+  return (
+    <ReachTooltip style={style} {...props}>
```

Comment:
Should we add the below class here ? as it is explicitly passed everywhere https://github.com/facebook/react/blob/327d5c48454aaee0d78fa69b7ef96a14605bf93b/packages/react-devtools-shared/src/devtools/views/Tooltip.css#L1-L13

Reply:
Would be a nice default then, yes.